Client/Frontend/Login Management/BreachAlertsManager.swift

```diff
@@ -44,11 +40,13 @@ final public class BreachAlertsManager {
 
         self.breachAlertsClient.fetchData(endpoint: .breachedAccounts) { maybeData in
             if maybeData.isSuccess, let data = maybeData.successValue {
-                let decoder = JSONDecoder()
-                if let decoded = try? decoder.decode([BreachRecord].self, from: data) {
-                    self.breaches = decoded
-                    completion(Maybe(success: self.breaches))
+                guard let decoded = try? JSONDecoder().decode([BreachRecord].self, from: data) else {
```

Comment:
Prefer `guard let data = maybe.successValue else { ... }` on line 42. 
Do you need to check maybe.isSuccess? I think that can be left out

Reply:
No, not needed. Will change to `guard` syntax in next commit